karma.conf.js

```diff
@@ -3,86 +3,89 @@
-      version: '54'
+    bs_chrome_sierra: {
+      base: 'BrowserStack',
+      browser: 'chrome',
+      browser_version: '64.0',
+      os: 'OS X',
+      os_version: 'Sierra'
     },
-    sl_chrome_35: {
-      base: 'SauceLabs',
-      browserName: 'chrome',
-      platform: 'Windows 7',
-      version: '31'
+    bs_ie11_win81: {
+      base: 'BrowserStack',
+      browser: 'ie',
+      browser_version: '11.0',
+      os: 'Windows',
+      os_version: '8.1'
     },
-    sl_firefox_37: {
-      base: 'SauceLabs',
-      browserName: 'firefox',
-      version: '37'
+    bs_ie10_win81: {
+      base: 'BrowserStack',
+      browser: 'ie',
+      browser_version: '10.0',
+      os: 'Windows',
+      os_version: '8'
     },
-    sl_firefox_31: {
-      base: 'SauceLabs',
-      browserName: 'firefox',
-      version: '31'
+    bs_ie9_win7: {
+      base: 'BrowserStack',
+      browser: 'ie',
+      browser_version: '9.0',
+      os: 'Windows',
+      os_version: '7'
     },
-    sl_ie_11: {
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 10',
-      version: '11'
+    bs_ie16_win10: {
+      base: 'BrowserStack',
+      browser: 'edge',
+      browser_version: '16.0',
+      os: 'Windows',
+      os_version: '10'
     },
-    sl_ie_10: {
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 7',
-      version: '10'
+    bs_safari_10_3: {
+      base: 'BrowserStack',
+      browser: 'Mobile Safari',
+      os: 'ios',
+      os_version: '10.3',
+      devices: ['iPhone SE']
     },
-    sl_ie_9: {
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 7',
-      version: '9'
+    bs_safari_11_iOS: {
+      base: 'BrowserStack',
+      browser: 'Mobile Safari',
+      os: 'ios',
+      os_version: '11.0',
+      real_devices: ['iPhone SE']
     },
-    sl_ie_8: {
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 7',
-      version: '8'
+    bs_safari_high_sierra: {
+      base: 'BrowserStack',
+      browser: 'Safari',
+      browser_version: '11.0',
+      os: 'OS X',
+      os_version: 'High Sierra'
     },
-    sl_safari_9: {
-      base: 'SauceLabs',
-      browserName: 'safari',
-      platform: 'OS X 10.11',
-      version: '9'
-    },
-    sl_ios_safari_8_4: {
-      base: 'SauceLabs',
-      browserName: 'iphone',
-      platform: 'OS X 10.10',
-      version: '8.4'
-    },
-    sl_ios_safari_9_1: {
-      base: 'SauceLabs',
-      browserName: 'iphone',
-      platform: 'OS X 10.10',
-      version: '9.1'
-    },
-    sl_android_5_0: {
-      base: 'SauceLabs',
-      browserName: 'android',
-      deviceName: 'Android Emulator',
-      platform: 'Linux',
-      version: '5.0'
+    bs_android_6: {
+      base: 'BrowserStack',
+      browser: 'android',
+      os: 'android',
+      os_version: '6.0',
+      device: 'Google Nexus 6',
+      real_mobile: true
     }
   };
```

Comment:
No IE8? (I'm not against dropping IE8 if usage numbers are finally low enough, but that should be a deliberate decision, as long as we claim we support it we should be testing on it)

Reply:
@SimonWoolf Fixing. See above.